src/main/java/nl/tudelft/lifetiles/graph/models/sequence/SequenceSegment.java

```diff
@@ -154,6 +160,110 @@ public final long distanceTo(final SequenceSegment other) {
      */
     @Override
     public final int compareTo(final SequenceSegment other) {
-        return Long.compare(this.getStart(), other.getStart());
+        int candidateComp = Long.compare(this.getAbsStart(),
```

Comment:
Could do that, but it doesn't make the code much easier to read, or shorter. Theres certainly no obfuscation, since its easy enough to read.